.github/workflows/events-nightly.yml

```diff
@@ -89,8 +89,10 @@ jobs:
       fail-fast: false
       matrix:
         # the config used in '.azure-pipelines/gpu-tests.yml'
-        python_version: ["3.7"]
-        pytorch_version: ["1.8"]
+        python_version: ["3.9"]
+        pytorch_version: ["1.8", "1.10"]
+        include:
+          - {python_version: "3.7", pytorch_version: "1.8"}
```

Comment:
Can we just keep the build for the image used by azure?

The less jobs we run the less opportunities for errors messing with a green CI.

Reply:
let's keep these three, I agree that we do not need a full spect, but we may use py3.9 soon

Reply:
Let's build 2 then, LTS and latest